lib/resolvers/project.js

```diff
@@ -71,7 +71,7 @@ module.exports = ({ models }) => ({ action, data, id, meta }, transaction) => {
               issueDate: startDate.toISOString(),
               expiryDate,
               title: version.data.title,
-              licenceNumber: project.licenceNumber || generateLicenceNumber()
+              licenceNumber: project.licenceNumber || generateLicenceNumber('project', project.schemaVersion > 0)
```

Comment:
I'd just pass the schema version here _just in case_ we ever have a v2 schema.

Reply:
done